Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -166,7 +166,14 @@ file that was distributed with this source code.
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
             {{- widget|raw -}}
-            {{- label is not same as(false) ? label|trans({}, translation_domain) -}}
+            {# if label is defined and not on embedded collection #}
+            {%- if label is not same as(false) -%}
+                {%- if name == '_delete' -%}
+                    <span class="sr-only">{{ label|trans({}, translation_domain) }}</span>
+                {%- else -%}
+                    {{ label|trans({}, translation_domain) }}
+                {%- endif -%}
+            {%- endif -%}
```

Comment:
Also it would be nice if this solution worked for all embedded checkboxes, not just `_delete`.
Although I don't really know if this doable...
Maybe with a custom function that walks up the form's ancestor and returns whether the current one is a child of `sonata_type_collection` ?
Anybody has an idea ?